### ISSUE TITLE
1.26.4 Timeless Node Auto-Highlight Issue

### ISSUE THREAD

**Issue by Cleista** [Lailloken/Lailloken-UI#78]
Sometimes the highlighting feature from clicking on a socket fails to highlight any nodes. 

I tried to check the shadow socket with the following jewel
```
Item Class: Jewels
Rarity: Unique
Brutal Restraint
Timeless Jewel
--------
Limited to: 1 Historic
Radius: Large
--------
Item Level: 86
--------
Denoted service of 6274 dekhara in the akhara of Asenath
Passives in radius are Conquered by the Maraketh
Historic
--------
They believed themselves the most ordered, but that tradition turned their forests to salt.
--------
Place into an allocated Jewel Socket on the Passive Skill Tree. Right click to remove from the Socket.
```
and
```
Alchemist's Genius
Attack + Cast Speed
F-Charge on Kill
Maximum Life
Non-Curse Aura Effect
Onslaught on Kill
```
selected

The application didn't highlight any nodes.

**Comment by Lailloken** [comment]
Thanks for reporting this bug and including the necessary information. It's caused by the 'infused' node: there's a system that checks for ambiguity when constructing the string for the in-game search field, so that similarly named nodes are not highlighted when they are not actually affected by a desired modification/override. I'll need to add a secondary check, which I'll do shortly.  

**Additional info on why this happens (if you're interested):**
The search field only accepts 100 characters, and if a socket hits a high number of desired mods/overrides, the entries for the string need to be truncated. If one of the affected nodes was, for example, a 'claws of the...' node and it was truncated to the point that the last word was omitted, all three of them (one is not in range of the jewel socket, but still) would be highlighted on the tree regardless of whether they're affected by a desired mod or not, which could be misleading.

In your case, the infused node triggers the ambiguity check because there's also the infused flesh node. That node is nowhere near, but I had to implement this ambiguity check globally because it's easier this way (I couldn't check every socket's surrounding notables for ambiguity, I would have gone insane). I chose to let the auto-highlighting fail in such cases because, otherwise, the user would need to check every auto-highlight themselves after clicking a socket to make sure it's actually correct. I need to override the ambiguity check when an entry in the string is not being truncated (as is the case with 'infused' in your case).
